src/multibody/geometry.hpp

```diff
@@ -447,15 +397,14 @@ struct GeometryObject
     GeometryData(const Data & data, const GeometryModel & model_geom)
         : data_ref(data)
         , model_geom(model_geom)
-        , oMg_collisions(model_geom.ncollisions)
-        , oMg_visuals(model_geom.nvisuals)
-        , oMg_fcl_collisions(model_geom.ncollisions)
+        , oMg_geometries(model_geom.ngeoms)
+        , oMg_fcl_geometries(model_geom.ngeoms)
         , collision_pairs()
         , nCollisionPairs(0)
         , distance_results()
         , collision_results()
     {
-      const std::size_t num_max_collision_pairs = (model_geom.ncollisions * (model_geom.ncollisions-1))/2;
+      const std::size_t num_max_collision_pairs = (model_geom.ngeoms * (model_geom.ngeoms-1))/2;
       collision_pairs.reserve(num_max_collision_pairs);
       distance_results.resize(num_max_collision_pairs, DistanceResult( fcl::DistanceResult(), 0, 0) );
```

Comment:
Should be "reserve" instead of "resize", isnt it? (both in :408 and :409)

This remark is not directly about the PR, but maybe it can be included.